basic literacy.i7x

```diff
@@ -7,110 +7,118 @@ Basic Literacy by Bart Massey begins here.
 
-Report writing it on (this is the report writing rule): say  "You write on [the second noun]."
+Report writing (this is the report writing rule): say  "You write on [the second noun]."
 
-Chapter - Erasing
 
-An erasing tool is a kind of thing.
+Chapter - Erasing
 
+A thing can be erasable. An erasing tool is a kind of thing. 
 To say error too many erasables (this is the say error several erasables rule): say "There are several things to erase."
-
 To say error no erasables (this is the say error no erasables rule): say "There is nothing to erase."
-
 Erasing is an action applying to one touchable thing and requiring light. Understand "erase" or "erase [thing]" as erasing.
 
-Rule for supplying a missing noun when erasing (this is the choose an erasable rule): let N be the number of touchable writable things; if N is greater than one, instead say error too many erasables; if N is zero, instead say error no erasables; now the second noun is a random touchable writable thing.
+Rule for supplying a missing noun when erasing (this is the choose an erasable rule): let N be the number of touchable writable things; if N is greater than one, instead say error too many erasables; if N is zero, instead say error no erasables; now the noun is a random touchable writable thing.
 
 To say error (T - a thing) need not be erased (this is the say error no writing rule): say "There is no writing on [the T] to erase."
-
-To say error (T - a thing) is not erasable (this is the say error not erasable rule): say "[The T] is not erasable."
-
+To say error (T - a thing) cannot be erased (this is the say error not erasable rule): say "[The T] is not erasable."
 To say error no erasing tool (this is the say error no erasing tool rule): say "You need something to erase with."
+To say error failed to obtain an erasing tool (this is the say error failed to obtain an erasing tool rule): say "You fail to find anything usable to erase with."
 
-Rule for implicitly taking an erasing tool (this is the implicitly take an erasing tool rule): let E be a random erasing tool; silently try taking E; say implicitly taking E.
-
-Check erasing (this is the check erasing rule): if the noun is not writable or the read text of the noun is empty, instead say error the noun need not be erased; if the noun is not writable, instead say error the noun is not erasable; if the player carries an erasing tool, rule succeeds; if an erasing tool is touchable, abide by the implicitly take an erasing tool rule; instead say error no erasing tool.
+Before erasing when the player does not carry an erasing tool and an erasing tool is touchable (this is the obtain an erasing tool rule):
+	repeat with E running through touchable erasing tools:
```

Comment:
Why do we have to try all the touchable erasing tools? In case one is nailed down or something?